setup.py

```diff
@@ -4,7 +4,7 @@
 
 requirements = [
         "requests",
-        "numpy",
+        "shapely",
```

Comment:
Note: this exposed a previously-suppressed linting warning in CodeFactor about a cyclic import. Not actually causing any problems at this time and absolutely not due to this PR, so I'm re-suppressing for the time being.